### ISSUE TITLE
<li> elements must be contained in a <ul> or <ol> - Results Page

### ISSUE THREAD

**Issue by davefollett** [codingblocks/podcast-app#225]
## 👣 Steps to reproduce
[Run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test) after performing a search.  Or go directly to:
* http://localhost:3000/search/vuejs

This issue will be flagged for each search result.

## ⚛️ Element(s) with issue
`<li class="sc-cSHVUG bMqOjD sc-jTzLTM fTgHXZ" data-item-type="search">`

## 💥 Impact rating
serious

## 🛠️ Recommended fix
List item does not have a `<ul>`, `<ol>` or role="list" parent element

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/listitem?application=AxeChrome
